akash1/HTML/ASSIn4/tableQ1.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>tableQ1</title>
+    <style>
+        table,
```

Comment:
why to add table, th, td in different lines just give style like table, th, td { style u want to give } like this